README.md

```diff
@@ -171,7 +184,19 @@ Getting Started - Build a Software-Interface
 3.  Write a C/C++ executable that interfaces with your design accordingly. To get a better understanding of this, you might want to refer to the collection of examples and the corresponding README which is located in `$TAPASCO_HOME/runtime/examples`
 4.  Build and Compile your Software.
 
-
+Using the Simulation
+--------------------
+1.  Design the Accelerator using HLS/HDL for the platform `sim`.
+2.  The TaPaSCo-Toolflow will generate a ZIP-file in place of a bitstream.
+3.  Load the design and start the simulation with `tapasco-start-sim path/to/zip`
```

Comment:
Please add a short description for the ``--gui`` and ``--unsafe-sim`` flags here.